src/neurostatslib/basis.py

```diff
@@ -64,7 +64,7 @@ def _evaluate(self, *xi: NDArray) -> NDArray:
         pass
 
     @staticmethod
-    def _get_samples(*n_samples: int) -> Generator[NDArray, ...]:
+    def _get_samples(*n_samples: int) -> Generator[NDArray, None, None]:
```

Comment:
what does this notation mean -- it's a generator returning an NDArray and what?

Reply:
those are the arguments of generator.send and generator.close; Since it is an overkill, I removed the two `None`, and the static checker doesn't complain